templates/leptos-ssr/metadata/snippets/component.txt

```diff
@@ -0,0 +1,20 @@
+[[trigger.http]]
+route = "{{http-path}}..."
```

Comment:
I recommend using the `http_wildcard` filter here - this will create the correct route regardless of whether the user entered `/foo/...`, `/foo/` or `/foo`

```suggestion
route = "{{http-path | http_wildcard}}"
```

Reply:
Ooh nice! That's much better, good call. Will update tonight